flake.nix

```diff
@@ -68,6 +68,7 @@
           drv = changelog;
         };
         apps.default = apps.changelog;
+        packages.default = changelog;
```

Comment:
I would create a `packages.changelog = changelog` and then point to that as a default, similar to the devshell/apps.